### PR TITLE
Add publish script for new website

### PR DESCRIPTION
**Pull request by rambleraptor**
This adds a new Github Actions workflow that triggers the site-generator-beta to publish whenever a new commit is made to main. site-generator-beta is responsible for cloning from main + publishing the new site.

A new organization secret will be required. I have the token but I need permission to add it to the org (preferably) or the repo

## 🍱 Types of changes

What types of changes does your code introduce to AEP? _Put an `x` in the boxes
that apply_

- [ ] Enhancement
- [ ] [New proposal](https://aep.dev/1#workflow)
- [ ] Migrated from google.aip.dev
- [ ] Chore / Quick Fix

## 📋 Your checklist for this pull request

Please review the [AEP Style and Guidance](https://aep.dev/style-guide) for
contributing to this repository.

### General

- [ ] Basic [Guidance](https://aep.dev/style-guide#guidance) is met.
- [ ] Ensure that your PR
      [references AEPs](https://aep.dev/style-guide#referencing-aeps)
      correctly.
- [ ] [My code has been formatted](https://aep.dev/contributing#formatting)
      (usually `prettier -w .`)

### Additional checklist for a new AEP

- [ ] A new AEP **should** be no more than two pages if printed out.
- [ ] Ensure that the PR is editable by maintainers.
- [ ] Ensure that [File structure](https://aep.dev/style-guide#file-structure)
      guidelines are met.
- [ ] Ensure that
      [Document structure](https://aep.dev/style-guide#document-structure)
      guidelines are met.

💝 Thank you!

### CODE REVIEW

File: .github/workflows/publish_site.yaml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
```
Comment (toumorokoshi) [inline .github/workflows/publish_site.yaml:1]: nit, but we already have https://github.com/aep-dev/aep.dev/blob/main/.github/workflows/publish.yaml. Is it worth joining this workflow with that one?

**Review by toumorokoshi — APPROVED** [review]
nice! will look into getting you access to upload the token.

**Comment by toumorokoshi** [comment]
@rambleraptor gave you all repository admin - let's follow up offline if you don't get the permissions you need from that.

File: .github/workflows/publish_site.yaml (hunk 1 above)
Comment (rambleraptor) [reply .github/workflows/publish_site.yaml:1]: I'd like to keep them separate so they kick off separate runners. We don't want one runner to interrupt another.

**Pull request merged**